src/java.base/share/classes/java/io/Console.java

```diff
@@ -390,13 +391,29 @@ public void flush() {
         pw.flush();
     }
 
+
+    /**
+     * Returns the {@code Charset} object used in this {@code Console}.
+     * <p>
+     * The returned charset corresponds to input and output source
+     * (e.g., keyboard and/or display) specified by the host environment or user.
+     * It may not necessarily be the same as the default charset returned from
+     * {@link java.nio.charset.Charset#defaultCharset() Charset.defaultCharset()}.
+     *
+     * @return A {@code Charset} object used in this {@code Console}.
+     * @since 17
```

Comment:
A couple of minor comments: 
May replace {@code Charset} with @link;
Add "the" to the sentence: The returned charset corresponds to "the" input...
@return: javadoc guide suggests "do not capitalize and do not end with a period" when writing a phrase. But I guess for consistency in this class, it's okay to capitalize.

Reply:
Thanks, Joe. Modified as suggested.

Reply:
Thanks. Looks good to me.